client/cmd/internal/plan/resource.go

```diff
@@ -0,0 +1,25 @@
+package plan
+
+import "strings"
+
+type ResourcePlan struct {
```

Comment:
Are we creating a plan for a single resource, or it should be an operation on an entity level.

Reply:
@sbchaos, I add plan result example in this PR https://github.com/goto/optimus/pull/228 description on section `Plan Result Example`

Reply:
will revisit this for the improvement, as it's not critical